<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReleasePage</class>
 <widget class="QWidget" name="ReleasePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="contentsStack">
     <widget class="QWidget" name="emptyPage">
      <layout class="QVBoxLayout" name="emptyPageLayout">
       <item>
        <widget class="QLabel" name="nothingFoundLabel">
         <property name="text">
          <string>Product not found</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="errorPage">
      <layout class="QVBoxLayout" name="errorPageLayout">
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="text">
          <string>An error occurred</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="loaderPage">
      <layout class="QVBoxLayout" name="loaderPageLayout">
       <item>
        <widget class="QProgressBar" name="loader">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resultsPage">
      <layout class="QVBoxLayout" name="resultsPageLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QWidget" name="header" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <layout class="QGridLayout" name="headerLayout">
          <item row="1" column="2" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
           <widget class="QLabel" name="userLabelsLabel">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignBottom">
           <widget class="QLabel" name="titleLabel">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">Release title</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignTop">
           <widget class="QLabel" name="platformLabel">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop">
           <widget class="QLabel" name="ratingLabel">
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="resultsStack">
         <widget class="QWidget" name="resultsOverviewPage">
          <layout class="QVBoxLayout" name="resultsOverviewPageLayout">
           <item>
            <widget class="QScrollArea" name="resultsOverviewScrollArea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="resultsOverviewScrollAreaContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>776</width>
                <height>450</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
               <item alignment="Qt::AlignmentFlag::AlignTop">
                <widget class="QWidget" name="resultsOverviewInfo" native="true">
                 <layout class="QVBoxLayout" name="resultsOverviewInfoLayout" stretch="0,0,0,0,0,0,0,0">
                  <item>
                   <widget class="QLabel" name="overviewTitleLabel">
                    <property name="text">
                     <string>Overview</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QScrollArea" name="mediaScrollArea">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>132</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>132</height>
                     </size>
                    </property>
                    <property name="verticalScrollBarPolicy">
                     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
                    </property>
                    <property name="widgetResizable">
                     <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="mediaScrollAreaContents">
                     <property name="geometry">
                      <rect>
                       <x>0</x>
                       <y>0</y>
                       <width>489</width>
                       <height>124</height>
                      </rect>
                     </property>
                     <layout class="QHBoxLayout" name="mediaScrollAreaContentsLayout">
                      <property name="spacing">
                       <number>24</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="descriptionLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="criticsLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="genreLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="themeLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="releaseDateLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="developerLabel">
                    <property name="text">
                     <string notr="true"/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="resultsOverviewStatsLayout"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="resultsProgressPage">
          <layout class="QVBoxLayout" name="resultsProgressPageLayout">
           <item>
            <widget class="QWidget" name="resultsProgressPageHeader" native="true">
             <layout class="QHBoxLayout" name="resultsProgressPageHeaderLayout">
              <item>
               <widget class="QLabel" name="achievementsIconLabel">
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources.qrc">:/icons/trophy.svg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="achievementsProgressLabel">
                <property name="text">
                 <string>Achievements N/A</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="achievementsRankLabel">
                <property name="text">
                 <string>Friends rank N/A</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gametimeIconLabel">
                <property name="maximumSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string notr="true"/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../resources.qrc">:/icons/clock.svg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gametimeLabel">
                <property name="text">
                 <string>Game time N/A</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="gametimeRankLabel">
                <property name="text">
                 <string>Friends rank N/A</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="resultsProgressPageScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="resultsProgressPageScrollContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>100</width>
                <height>30</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="resultsProgressPageScrollContentsLayout"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="resultsExtrasPage">
          <layout class="QVBoxLayout" name="resultsExtrasPageLayout">
           <item>
            <widget class="QScrollArea" name="resultsExtrasPageScrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="resultsExtrasPageScrollAreaContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>100</width>
                <height>30</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
